velero/backup/common-service/label-common-service.sh

```diff
@@ -0,0 +1,184 @@
+    namespaces=$(${OC} get configmap namespace-scope -n $OPERATOR_NS -oyaml | awk '/^data:/ {flag=1; next} /^  namespaces:/ {print $2; next} flag && /^  [^ ]+: / {flag=0}')
+    # add cert-manager namespace and licensing namespace and lsr namespace into the list with comma separated
+    namespaces+=",$CONTROL_NS,$CERT_MANAGER_NAMESPACE,$LICENSING_NAMESPACE,$LSR_NAMESPACE"
+    namespaces=$(echo "$namespaces" | tr ',' '\n')
+
+    while IFS= read -r namespace; do
+        # Label the namespace
+        ${OC} label namespace "$namespace" foundationservices.cloudpak.ibm.com=namespace --overwrite=true 2>/dev/null
+        
+        # Label the OperatorGroup
+        operator_group=$(${OC} get operatorgroup -n "$namespace" -o jsonpath='{.items[*].metadata.name}')
+        ${OC} label operatorgroup "$operator_group" foundationservices.cloudpak.ibm.com=operatorgroup -n "$namespace" --overwrite=true 2>/dev/null
+        
+        # Label the entitlement key
+        ${OC} label secret ibm-entitlement-key foundationservices.cloudpak.ibm.com=entitlementkey -n "$namespace" --overwrite=true 2>/dev/null
+        
+        # Label the OperandRequest
+        operand_requests=$(${OC} get operandrequest -n "$namespace" -o custom-columns=NAME:.metadata.name --no-headers)
+        # Loop through each OperandRequest name
+        while IFS= read -r operand_request; do
+            ${OC} label operandrequests $operand_request foundationservices.cloudpak.ibm.com=operand -n "namespace" --overwrite=true 2>/dev/null
+        done <<< "$operand_requests"
+    done <<< "$namespaces"
+
+    ${OC} label secret ibm-entitlement-key foundationservices.cloudpak.ibm.com=entitlementkey -n $DEFAULT_SOURCE_NS --overwrite=true 2>/dev/null
+    ${OC} label secret pull-secret -n openshift-config foundationservices.cloudpak.ibm.com=pull-secret --overwrite=true 2>/dev/null
+
+}
+
+function label_configmap() {
+    
+    title "Start to label the ConfigMaps... "
+    ${OC} label configmap common-service-maps foundationservices.cloudpak.ibm.com=configmap -n kube-public --overwrite=true 2>/dev/null
+    ${OC} label configmap cs-onprem-tenant-config foundationservices.cloudpak.ibm.com=configmap -n $SERVICES_NS --overwrite=true 2>/dev/null
+    ${OC} label configmap platform-auth-idp foundationservices.cloudpak.ibm.com=configmap -n $SERVICES_NS --overwrite=true 2>/dev/null
+}
+
+function label_subscription() {
+
+    title "Start to label the Subscriptions... "
+    local cs_pm="ibm-common-service-operator"
+    local cm_pm="ibm-cert-manager-operator"
+    local lis_pm="ibm-licensing-operator-app"
+    local lsr_pm="ibm-license-service-reporter-operator"
+    
+    ${OC} label subscriptions.operators.coreos.com $cs_pm foundationservices.cloudpak.ibm.com=subscription -n $OPERATOR_NS --overwrite=true 2>/dev/null
+    ${OC} label subscriptions.operators.coreos.com $cm_pm foundationservices.cloudpak.ibm.com=subscription -n $CERT_MANAGER_NAMESPACE --overwrite=true 2>/dev/null
```

Comment:
should be singleton-subscription for the label for cert manager and licensing and lsr